doc/contribute/build_ccf.rst

```diff
@@ -59,7 +59,19 @@ Tests can be started through the ``tests.sh`` wrapper for ``ctest``.
 .. code-block:: bash
 
     $ cd build
-    $ ./tests.sh
+    $ ./tests.sh -LE partitions
+
+The `partitions` test requires `root` access (to set up firewall rules), so it will not pass when using normal user accounts.
```

Comment:
IMO a better way to do that is to have a check at the top of the partition test that prints this message when you try to run it. I expect nobody to find this piece of documentation before that warning.

Reply:
Well, docs don't need to be before the fact, they can also be reactive (`what did I do wrong?`).

But I prefer @eddyashton suggestion to not have that problem in the first place.